proto/rill/runtime/v1/queries.proto

```diff
@@ -170,6 +170,14 @@ service QueryService {
     };
   }
 
+  // ResolveMetricsViewFilterExpression converts a filter expression to a SQL filter string.
+  rpc ResolveMetricsViewFilterExpression(ResolveMetricsViewFilterExpressionRequest) returns (ResolveMetricsViewFilterExpressionResponse) {
```

Comment:
nit: can we call this RPC `ConvertExpressionToMetricsSQL` or something like that? To clarify that it's more a conversion than a resolution (i.e. it doesn't execute/expand the expression)